### ISSUE TITLE
Sending message to Twitter is broken

### ISSUE THREAD

**Issue by pascalberger** [cake-contrib/Cake.Recipe#245]
```
Sending message to Twitter...
System.InvalidOperationException: Misused header name. Make sure request headers are used with HttpRequestMessage, response headers with HttpResponseMessage, and content headers with HttpContent objects.
   at System.Net.Http.Headers.HttpHeaders.CheckHead
erName(String name)
   at System.Net.Http.Headers.HttpHeaders.Add(String name, String value)
   at Cake.Twitter.TwitterProvider.SendTweet(String message)
   at Submission#0.TwitterSendTweet(String oAuthConsumerKey, String oAuthConsumerSecret, String accessToken, String accessTokenSecret, String message)
   at Submission#0.SendMessageToTwitter()
```

See https://ci.appveyor.com/project/cakecontrib/cake-issues/build/0.4.1.build.137#L840

**Comment by gep13** [comment]
This is related to this: https://github.com/cake-contrib/Cake.Twitter/issues/30

**Comment by gep13** [comment]
This is fixed in 0.9.0 of Cake.Twitter.
